dlt/destinations/duckdb/duck.py

```diff
@@ -41,6 +47,24 @@
 }
 
 
+class DuckDbCopyJob(LoadJob):
+    def __init__(self, table_name: str, write_disposition: TWriteDisposition, file_path: str, sql_client: DuckDbSqlClient) -> None:
+        super().__init__(FileStorage.get_file_name_from_file_path(file_path))
+        qualified_table_name = sql_client.make_qualified_table_name(table_name)
```

Comment:
This is always referencing the non-staging table, right? Should be done inside `with_staging_dataset` context.  
But the merge test runs so I'm not sure if this is right.

Reply:
I removed the merge with `replace` write disposition if you look closer at test.